packages/ilib-loctool-webos-c/CFile.js

```diff
@@ -120,9 +120,8 @@ CFile.prototype.parse = function(data) {
     // To extract resBundle_getLocString()
```

Comment:
I would consider changing the RE on line 103 to:

```
var reGetLocString = new RegExp(/\bresBundle_getLocString\(\s*([^"][^,]*|"(\\"|[^"])*")\s*\,\s*"((\\"|[^"])*)"\s*\)/g);
```

This has the extra "ungreedy" operators on it, for safety.

Reply:
The current regular expression is the same as you suggested. and I realized the issue has been resolved only with the slight modification.